fastlane/spec/plugin_manager_spec.rb

```diff
@@ -2,12 +2,12 @@
   describe Fastlane::PluginManager do
     describe "#gemfile_path" do
       it "returns an absolute path if Gemfile available" do
-        expect(Fastlane::PluginManager.new.gemfile_path).to eq(File.expand_path("Gemfile"))
+        expect(Fastlane.plugin_manager.gemfile_path).to eq(File.expand_path("Gemfile"))
```

Comment:
I think it is better for unit tests not to involve any global variables, especially if they cache instances. This might lead to unexpected results where state is retained across tests.